sdk/tables/azure-data-tables/src/policies/switch_to_secondary_policy.cpp

```diff
@@ -5,15 +5,15 @@
 
 namespace Azure { namespace Data { namespace Tables { namespace _detail { namespace Policies {
 
-  Azure::Core::Context::Key const SecondaryHostReplicaStatusKey;
+  Azure::Core::Context::Key const SecondaryHostReplicaStatus;
```

Comment:
This isn't the actual status, but rather the context key to get the status value. We should consider a different name, but given this is an implementation detail, it can be fixed separately.

Reply:
Was this causing a name-collision? It's a different namespace. Do we need to change this?